digital_asset_types/src/tests/get_assets_by_owner.rs

```diff
@@ -1,322 +1,316 @@
 #[cfg(test)]
-mod get_assets_by_owner {
-    use sea_orm::{
-        entity::prelude::*, Condition, DatabaseBackend, JoinType, MockDatabase, QuerySelect,
-    };
-    use solana_sdk::{signature::Keypair, signer::Signer};
+use sea_orm::{entity::prelude::*, DatabaseBackend, MockDatabase};
+use solana_sdk::{signature::Keypair, signer::Signer};
 
-    use crate::{
-        adapter::{Creator, TokenProgramVersion, TokenStandard},
-        dao::{
-            asset, asset_authority, asset_creators, asset_data,
-            prelude::AssetData,
-            sea_orm_active_enums::{OwnerType, RoyaltyTargetType},
-        },
-        tests::{
-            create_asset, create_asset_authority, create_asset_creator, create_asset_data,
-            MetadataArgs,
-        },
-    };
+use crate::{
+    adapter::{Creator, TokenProgramVersion, TokenStandard},
+    dao::{
+        prelude::AssetData,
+        {asset, asset_authority, asset_creators, asset_data},
+        sea_orm_active_enums::{OwnerType, RoyaltyTargetType},
+    },
+    tests::{
+        create_asset, create_asset_authority, create_asset_creator, create_asset_data, MetadataArgs,
+    },
+};
 
-    #[cfg(feature = "mock")]
-    #[tokio::test]
-    async fn get_assets_by_owner() -> Result<(), DbErr> {
-        let id_1 = Keypair::new().pubkey();
-        let owner_1 = Keypair::new().pubkey();
-        let update_authority_1 = Keypair::new().pubkey();
-        let creator_1 = Keypair::new().pubkey();
-        let uri_1 = Keypair::new().pubkey();
+#[async_std::test]
```

Comment:
why switch from `tokio` to `async_std`?  Just curious.

Reply:
the testing setup changed when it was imported to this repo. so i did some refactoring according to seaorm testing docs